packages/mwp-app-route-plugin/src/route.js

```diff
@@ -13,13 +13,241 @@ export const onPreResponse = {
 	method: (request: HapiRequest, reply: HapiReply) => {
 		const response = request.response;
 		if (!response.isBoom || process.env.NODE_ENV === 'production') {
-			return reply.continue();
+			return reply(`
+<!DOCTYPE html>
+<html data-reactroot="">
+
+<head>
+	<title data-react-helmet="true">Meetup</title>
+	<meta data-react-helmet="true" name="viewport" content="width=device-width, initial-scale=1" />
+	<meta data-react-helmet="true" http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<meta data-react-helmet="true" http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
might want to keep the markup as simple as possible - it doesn't need to be optimized for _anything_ other than fast render